src/main/java/com/sonyericsson/rebuild/RebuildAction.java

```diff
@@ -487,4 +489,18 @@ public RebuildParameterPage getRebuildParameterPage(ParameterValue value) {
         // So Jelly fallback could occur.
         return null;
     }
+
+    /**
+     * Method for copying of old build parameters (such as git revision from git plugin)
+     *
+     * @param build old build for parameters extraction
+     * @param actions list for filling
+     */
+    private void fillOtherBuildActions(Run build, List<Action> actions) {
+        for (Action a: build.getActions()) {
+            if (!(a instanceof CauseAction) && !(a instanceof ParametersAction)) {
+                actions.add(a);
```

Comment:
It's really unsafe, because there are many plugins contributing actions as metadata.
Such copying may potentially cause a duplication of data with errorneous results in plugins